### PR TITLE
resync changelog

### PR DESCRIPTION
**Pull request by patricoferris**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- Add code-highlighting to tutorials (#108, by @patricoferris)
+- Add initial set of Best Known Methods (#107, by @tmattio)
+- Add initial toplevel to homepage (#106, by @tmattio)
+- Serve HTML pages from server (#101, by @tmattio)
```
Comment (tmattio) [inline CHANGES.md:6]: This is not a user-facing change, so I'm not sure it should be included in the changelog (same for the previous items "Merge frontend" and "Merge ood")

**Review by tmattio — APPROVED** [review]
Thanks @patricoferris! We could add a check in the pipeline for a changelog entry also. (e.g. https://github.com/ocamllabs/vscode-ocaml-platform/blob/master/.github/workflows/changelog.yml)

Not sure whether the non-user-facing changes should be in the changelog, but if you think they should be in there, this LGTM.

**Comment by patricoferris** [comment]
@tmattio any idea why the check says https://github.com/ocaml/v3.ocaml.org-server/runs/3686040719#step:3:6 ?

**Comment by tmattio** [comment]
Not sure... You might need to fetch the repo maybe?

**Comment by patricoferris** [comment]
Hmm maybe, looks like vscode-platform has the same issue? https://github.com/ocamllabs/vscode-ocaml-platform/runs/3570923719?check_suite_focus=true#step:3:6

**Comment by patricoferris** [comment]
Yeah you were right, the checkout only gets exactly the ref/commit it needs and nothing else by default it seems. `fetch-depth: 0` gets *everything*. I think this is fixed now (https://github.com/ocaml/v3.ocaml.org-server/runs/3698943717) you might want to upstream that to vscode @tmattio ?

**Pull request merged**